src/olympia/versions/models.py

```diff
@@ -236,6 +236,23 @@ def delete(self, hard=False):
             self.save()
 
     @property
+    def is_user_disabled(self):
+        return self.files.filter(status=amo.STATUS_DISABLED).exclude(
+            original_status=amo.STATUS_NULL).count()
+
+    @is_user_disabled.setter
+    def is_user_disabled(self, disable):
+        # User wants to disable (and the File isn't already).
+        if disable:
+            for f in self.files.exclude(status=amo.STATUS_DISABLED).all():
```

Comment:
Komodo highlights `file` like a reserved word so I avoided it.  But sure, can do.  (Are you sure about that pep8 rule on variable length? - I've never seen it and it doesn't fail the flake8 travis run)